### PR TITLE
Improve svg import #542

### LINKED ISSUE #542: Find a better way to import svg icons.
Now we are loading svg icon like this:

```
 import accuracyIcon from '../../images/icons/accuracy.svg';
<img src={accuracyIcon} alt="accuracy" />
```

It's not so ideal, because as img's src, we can not do control over the SVG, like change the color by `fill` property. According to the material UI, we can use iconSVG or svgIcon, to do this, we failed to do it following the guide, please help us solve this problem.

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by jeremydthomas**
# Description

[comment]: # We can now use svg files as components I.E ```<AccuracyIcon />``` or use ```<SvgIcon component={AccuracyIcon} />``` .  All of the props available in MUI documentation work. I also changed all of the old way people did to use SVG files into the new way. 

Fixes #542 

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

# How Has This Been Tested?

visually checked to make sure all of my changes registered over

# Checklist:

- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by RubenSmn — APPROVED** [review]

**Comment by dadiorchen** [comment]
@jeremydthomas thanks for the contribution, good job. 

I downloaded and ran your pr on my computer, I found some icon appearance changed, can you look into  it? 

Screen Shot 2022-08-01 at 10 05 44 AM
Screen Shot 2022-08-01 at 10 02 35 AM

**Comment by dadiorchen** [comment]
Maybe you lost the css property on the previous icon to adjust the icon?

We need to be strict to respect the Figma design.

**Comment by dadiorchen** [comment]
BTW, the small the PRs are, the better.

For example, if you first give a PR for just the new approach of SVG file, then I can merge that one now without block by other problems.

**Comment by dadiorchen** [comment]
@jeremydthomas another question, why we need to replace previous svg file?

File: src/pages/organizations/[organizationid].js [hunk 1]
```diff
@@ -190,14 +194,14 @@ export default function Organization(props) {
...
-                  iconURI={calendarIcon}
+                  iconURI={CalendarIcon}
                   info={`Organization since ${moment().format(
                     'MMMM DD, YYYY',
                   )}`}
                 />
               </Box>
               <Box sx={{ mt: 2 }}>
-                <Info iconURI={locationIcon} info="Shirimatunda, Tanzania" />
+                <Info iconURI={LocationIcon} info="Shirimatunda, Tanzania" />
```
Comment (dadiorchen) [inline src/pages/organizations/[organizationid].js:204]: @jeremydthomas  for example, here, you put into the SVG to replace the `src`, but in the component:

```
    <Box
      sx={{
        color: 'text.text2',
        display: 'flex',
        alignItems: 'center',
        gap: 3,
        '& img': {
          filter: 'opacity(0.5)',
          maxWidth: 16,
          maxHeight: 16,
        },
      }}
    >
      <SvgIcon component={iconURI} inheritViewBox alt="join time" />
      <Typography variant="h6">{info}</Typography>
    </Box>
```

The `& img` property don't work anymore, so we need to make sure all these replacement is strictly equal to the previous icon. We did a lot of effort the make the UI perfect, only then we can get a beautiful app.

**Comment by dadiorchen** [comment]
@jeremydthomas feel free to huddle me on slack, and these days I will be on the huddle on engineering channel for couple of hours, so welcome.

I have realized your situation of being not able to split them to PRs.

Yes, I think we have to go through all these cases one by one and fix the impact on them. We can switch between `beat` and `your branch` and check the icons and fix them.

**Comment by dadiorchen** [comment]
@jeremydthomas this is a solid contribution, really appreciate it! Let's work together to finish it.

**Comment by dadiorchen** [comment]
I realized another benefits fo this solution is that it change the case from image file resource to bundled js resource, the advantage of later is that in the production env, we have to do some tweak on the URL to solve the path to point to the correct resource file (image file), but the later doesn't have to do it.

**Comment by jeremydthomas** [comment]
@dadiorchen I have 2 more work days, then I'll have time to do a huddle. I'll message you on slack when I can!

**Pull request merged**